crates/matrix-sdk-indexeddb/src/crypto_store/migrations.rs

```diff
@@ -317,6 +317,11 @@ async fn prepare_data_for_v7(serializer: &IndexeddbSerializer, db: &IdbDatabase)
         }
     }
 
+    // We have finished with the old store. Clear it, since it is faster to
+    // clear+delete than just delete. See https://www.artificialworlds.net/blog/2024/02/01/deleting-an-indexed-db-store-can-be-incredibly-slow-on-firefox/
```

Comment:
This link is a 404 :-).

Reply:
Seems like the author of the blog is to blame: https://www.artificialworlds.net/blog/2024/02/02/deleting-an-indexed-db-store-can-be-incredibly-slow-on-firefox/.